js/spreadsheet.js

```diff
@@ -531,6 +531,26 @@ var spreadsheet = function(options) {
 
       colWidths = getColWidths();
     },
+    beforeColumnMove: function(items, index) {
+      colWidths = getColWidths();
+
+      var length = items.length;
+      var colsToMove = colWidths.splice(items[0], length);
```

Comment:
any chance `items` can be empty here?

Reply:
> any chance `items` can be empty here?

the hook "beforeColumnMove" is activated when at least one item is selected